portfolio/src/main/webapp/index.html

```diff
@@ -7,13 +7,98 @@
+        target="_blank" class="link1">
+        <button class="linkedIn">LinkedIn</button>
+      </a>
+      <a href="https://github.com/mumair989"
+      target="_blank">
+      <button class="gitHub">GitHub</button>
+    </a>
+    <a href="#contact"><button class="Contact">Contact</button></a>
+    <a href="#projects"><button class="ProjectButton">Projects</button></a>
+    <a href="#secondBox"><button class="WorkButton">Experience</button></a>
+
+      
+      <!-- <p id="heading" >Hi there!</p> -->
+      <div id="image1">
+      <img src="Images/image1.PNG" alt="My picture" width="250" height="350" />
+      </div>
+<p id="sideNote"><i style="color:rgb(255, 0, 0)">Hi their!</i><br> I am Muhammad Umair <br> </p>
+
+<article id="about">
+  <h2 id="aboutHeader">About Me</h2>
+      <p id="myLife">My name is <i style="color:red;">Muhammad Umair</i>, I live in New York City, I am really excited to be a participant of Google SPS Program.
+        I joined Google SPS because I wanted to meet people from across the world and learn new things from everyone and the program itself.
+        I believe SPS will further develop my technical and interpersonal skills and hopefully I will create something really cool together.</p>
+      </article>
+     
+        <div id="projects">
+        <h2 id="projectHeader">Projects</h2>
+        <div id="chessBoard">
+        <img src="Images/chessBoard.jpeg" alt="ChessBoard picture" width="200" height=150" />
+    </div>
+    <h4 id="queenTitle">Fancy 8 Queen</h4>
+    <p id="queen">Created fancy chessboard where by running the code, prints out 92 solutions of a game w
+        ith 8 queens displayed on the board. Made with C++.<br><button id="8QueenCode">Code</button></p >     
+
+            <div id="ImageEditor">
+            <img src="Images/ImageEditor.png" alt="ImageEditor" width="200" height=150" />
+        </div>
+        <h4 id="editorTitle">ImageEditor</h4>
+        <p id="editorInfo">Created an ImageEditor Application that allows user to upload an image and edit it however they desire and also allows them to download into their local computer<br>
+             <a href="https://cssifinal.glitch.me/" target="_blank"><button>Demo</button> </a></p
+    </div>
+
+
+<div id="secondBox">
+    <h2 id="Work">Word Experience</h2>
+    <h4 id="google">Google(CSSI)</h4>
+    <p id="google-info">Participated in a 4-week intensive computer science summer program for high-achieving students
+        • Completed an introductory project-based java script processing curriculum taught by Google engineers
+        • Configured 25 individual coding projects by using concepts such as variables, data types, and functions
+        • Attended product design, resume development, and software engineering interview workshops
+        • Delivered a collaborative final project presentation that included a live demonstration to Google employees and community leaders</p>
```

Comment:
BTW, if you want this to actually appear as a bulleted list, consider using a [`<ul>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul) element instead